modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -3,19 +3,14 @@
     <AposButton
       type="rich-text"
       @click="click"
-      :class="{ 'apos-active': buttonActive }"
+      :class="{ 'is-active': buttonActive }"
```

Comment:
This complicates directions for people so they avoid our namespace.

Reply:
this pattern is everywhere in a3

Reply:
Yeah I'll open a ticket to fix this globally rather than slowing things down here.